packages/components/psammead-radio-schedule/src/index.stories.jsx

```diff
@@ -59,3 +63,38 @@ stories.add(
     ),
   { notes },
 );
+
+storiesOf('Components|RadioSchedule/StartTime', module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob())
+  .add(
+    'Timestamp',
+    ({ locale, script, service }) => {
+      return (
+        <StartTimestamp
+          timestamp={storiesUnixTimestamp}
+          timezone="Europe/London"
+          locale={locale}
+          script={script}
+          service={service}
+        />
+      );
+    },
+    { notes },
+  )
+  .add(
+    'Start time',
```

Comment:
```suggestion
    'default',
```
I think this should be the first story displayed.